### PR TITLE
bump whereabouts to v0.5.4

### PR DESCRIPTION
**Pull request by zhouke1991**
## What this PR does / why we need it
bump whereabouts to v0.5.4
https://github.com/k8snetworkplumbingwg/whereabouts/releases/tag/v0.5.4

### CODE REVIEW

File: addons/packages/whereabouts/0.5.4/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.15
+RUN mkdir -p $GOPATH/src/github.com/k8snetworkplumbingwg/whereabouts
+WORKDIR $GOPATH/src/github.com/k8snetworkplumbingwg/whereabouts
+RUN git clone -b v0.5.1 --single-branch https://github.com/k8snetworkplumbingwg/whereabouts.git .
```
Comment (12345lcr) [inline addons/packages/whereabouts/0.5.4/Dockerfile:4]: Please check if the version is correct

File: addons/packages/whereabouts/0.5.4/test/Makefile [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+ROOT_DIR := $(shell git rev-parse --show-toplevel)
+VERSION ?= 0.5.1
```
Comment (12345lcr) [inline addons/packages/whereabouts/0.5.4/test/Makefile:5]: Please check if the version is correct

File: addons/packages/whereabouts/0.5.4/test/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# WHEREABOUTS E2E TESTS
+
+## Prerequisites
+
+* A Tanzu Community Edition cluster and the cluster needs to be the
+  current-context. See the [Getting Started
+  Guide](https://tanzucommunityedition.io/docs/getting-started/) for
+  instuctions on how to create a cluster.
+* The packages `multus-cni.community.tanzu.vmware.com`  with version `3.7.1`
+  and `whereabouts.community.tanzu.vmware.com` with version `0.5.1` must
```
Comment (12345lcr) [inline addons/packages/whereabouts/0.5.4/test/README.md:10]: Please check if the version is correct

**Review by 12345lcr — APPROVED** [review]

**Review by wjun — COMMENTED** [review]
LGTM

**Review by wjun — APPROVED** [review]

**Review by seemiller — APPROVED** [review]

**Pull request merged**